install/terminal/mise.sh

```diff
@@ -1,7 +1,11 @@
+if [[ -z $UTILS_SOURCED ]]; then
+    source ~/.local/share/omakub/utils.sh
+fi
+ARCH=$(get_arch)
```

Comment:
```suggestion
ARCH=$(get_arch "mise")
```
And an entry in utils.sh as well.

Reply:
`get_arch` at here:  https://github.com/IbrahimTanyalcin/omakub/blob/raspi5-compat/utils.sh#L80-L94 will default to `lscpu`'s output if no entries are found here: https://github.com/IbrahimTanyalcin/omakub/blob/raspi5-compat/utils.sh#L37-L54

I think you want it to be more explicit, in that case I can add these indentity entries to the assoc array here: https://github.com/IbrahimTanyalcin/omakub/blob/raspi5-compat/utils.sh#L38 like: 

```shell
["mise.aarch64"]="aarch64"
["mise.x86_64"]="x86_64"
```

I only added values to that array for the programs that deviate from lscpu's standard arch notation. But if you believe being explicit is better I will do that. What do you think?